.browserslistrc

```diff
@@ -0,0 +1 @@
+last 1 version, >2%
```

Comment:
is there a reason why we are dropping firefox esr

Reply:
Check https://github.com/bigcommerce/cornerstone/pull/1836#discussion_r487841227